examples/templates/vue/src/App.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div id="example">
+    <hot-table :settings="hotSettings"></hot-table>
+  </div>
+</template>
+
+<script>
+import Handsontable from "handsontable";
+import { HotTable } from "@handsontable/vue";
+import Vue from "vue";
+import "handsontable/dist/handsontable.full.css";
+import './assets/styles.css';
```

Comment:
Different quote styles. We should add linter for code snippets in the next iteration :)

Reply:
That's 100% true! I hope we will get to it in the future

Reply:
Done https://github.com/handsontable/handsontable/pull/7588/commits/018c9186fa0a298d300df26586014926bb782d71